lychee-bin/src/options.rs

```diff
@@ -319,6 +348,11 @@ and 501."
     #[serde(default = "max_retries")]
     pub(crate) max_retries: u64,
 
+    // Minimum TLS Version
```

Comment:
Sorry for not noticing sooner, but if you change this to `///` then it will become part of the documentation. (I noticed because I wondered why the argument in the updated `README.md` didn't contain a help message.)

```suggestion
    /// Minimum TLS Version
```